com.unity.render-pipelines.universal/Editor/ShaderGUI/ShadingModels/LitGUI.cs

```diff
@@ -103,6 +126,44 @@ public static void Inputs(LitProperties properties, MaterialEditor materialEdito
                 materialEditor.TexturePropertySingleLine(Styles.occlusionText, properties.occlusionMap,
                     properties.occlusionMap.textureValue != null ? properties.occlusionStrength : null);
             }
+
+            DoClearCoat(properties, materialEditor, material);
+        }
+
+        public static bool ClearCoatEnabled(Material material)
```

Comment:
add public api docs or make api internal if it's not meant for public.

Reply:
Turned into private. Will push soon.